Assets/VRTK/Source/Scripts/Interactions/Interactables/GrabAttachMechanics/VRTK_RotateTransformGrabAttach.cs

```diff
@@ -278,6 +278,7 @@ public virtual void SetRotation(float newAngle, float transitionTime = 0f)
             }
             else
             {
+				CancelUpdateRotation();
```

Comment:
You should move this to https://github.com/thestonefox/VRTK/pull/1774/files#diff-2c617bc088c6541ec6c8322f2c6c5915R273 and take it out of the `true` condition as well.

also, you are missing an extra space indentation here

Reply:
Good point. Ok